packages/astro/test/fixtures/astro-markdown-shiki/default-color/astro.config.mjs

```diff
@@ -0,0 +1,17 @@
+const serendipity = JSON.parse(
```

Comment:
Do we need this large theme config for the test? Could we use a simpler one instead?